src/lib.rs

```diff
@@ -47,9 +54,24 @@ pub struct Args {
     pub vimgrep: bool,
 }
 
+impl Args {
+    pub fn use_paths(&self) -> Vec<PathBuf> {
```

Comment:
This mimics what I saw in `ripgrep`, I think I misunderstood the `WalkBuilder::new()` argument and thought it was for the "current directory" but no `ripgrep` just passes it the first of its paths (which like this it ensures at least includes `./` as a default) and then explicitly `.add()`'s the rest of those paths